### DISCUSSION TITLE
Intro/Mission Briefings Missing

### DISCUSSION THREAD

**Discussion by ghost** [ihhub/fheroes2#4760] [Q&A]
Sorry if this question has been asked a dozen times over. I'm a complete newcomer to Linux, and after managing to stagger through setting up fheroes2, I noticed that both the introduction cinematics, as well as the mission briefings, are missing. Is this normal, and they simply haven't been implemented, or did I do something wrong?

**Comment by ihhub** [comment]
Hi @Auraculum2 , welcome!

Missing videos mean missing files for them. Please check that you have **anim** directory within the same level where **data** directory is located.

**Reply by ghost** [reply]
Hi there!
I used "innoextract" to unpack my GOG version of the game, and I didn't have an ANIM folder within, only MUSIC, SOUND, DATA, and MAPS.

**Reply by ihhub** [reply]
Please use `extract_homm2_resources.sh` script for this. It comes in a compiled package.

**Reply by ghost** [reply]
"Bsdtar not found in your system. Please install it and re-run this script to extract animation resources." 

And when I try to insert "sudo apt-get install bsdtar" I receive "Package 'bsdtar' has no installation candidate."

**Reply by ihhub** [reply]
Perhaps @oleg-derevenetz  can advice here.

**Reply by ghost** [reply]
I run Pop!_OS 21.04, if this will be of any help.

**Reply by oleg-derevenetz** [reply]
Hi all,

Honestly, I never heard of such a distro before :) but `bsdtar` is in general a part of the `libarchive`, so may be installing a `libarchive` package may help.

**Reply by ghost** [reply]
That did it! Game is running with sound, as well as the intro/briefings. Just a few follow-up questions:
1. Is there any way to launch the game just through the heroes2 icon, instead of always launching the script through the terminal?
2. Is there a way to force the game to launch itself already stretched to fullscreen?
3. I don't know whether it's a bug, or something on my end, but during the briefings the crystal on the table is supposed to glow when Roland/Archibald speaks. It does not.

**Reply by oleg-derevenetz** [reply]
> Is there any way to launch the game just through the heroes2 icon, instead of always launching the script through the terminal?

It depends on your window manager. I suppose, all window managers should be able to create shortcuts, just in a different way.

> Is there a way to force the game to launch itself already stretched to fullscreen?

Once you switched to the fullscreen mode (using F4 button) this should be rememebered in the fheroes2.cfg and app should be launched in the fullscreen mode next time.

> I don't know whether it's a bug, or something on my end, but during the briefings the crystal on the table is supposed to glow when Roland/Archibald speaks. It does not.

This is a known bug: #4504
